CLI arguments

fsabuhi opened the pull request:
I added python's built-in argparser library to your code and did some changes in code 

I added two arguments as I only saw 2 input prompts in the code

Example:
```bash
$ python3 winLabeler.py --path "C:\Users\User\summer_photos\example.png" large
```
I currently don't use windows so I couldn't test it properly. Please test and share your thoughts with me so we can improve this 

 Issue #1

fsabuhi committed:
Update README.md

fsabuhi committed:
Added python argparse library to handle cli arguments

moxwel commented:
Hi, and thanks for your contribution.

I have tested this and works great! Although I should have been more specific in the first issue, my bad.

I have added a comment in #1 with more detail.

fsabuhi committed:
Update README.md

fsabuhi committed:
Added output_file_name argument

Deleted default arguments for small_folder and output_file name because winLabeler.py always passes value to them

fsabuhi committed:
Added -ns and -o flags

fsabuhi commented:
I added all arguments you mentioned on issue #1
Path argument doesn't require a flag and -ns, -o flags are optional with default values "False" and "out.ico".
Please test it and let's fix if there are any problems with it

fsabuhi committed:
Quick Fix
